src/os/ObjectStore.h

```diff
@@ -1523,6 +1523,10 @@ class ObjectStore {
   virtual bool wants_journal() = 0;  //< prefers a journal
   virtual bool allows_journal() = 0; //< allows a journal
 
+  virtual PerfCounters *get_perf_counter() {
```

Comment:
Why not implementing this method for FileStore and KStore? They have corresponding logger member too